Terminal Vignette

gtritchie opened the pull request:
Authored terminal API vignette, and included results of `pkgdown::build_site()` (not sure if I was supposed to do that).

kevinushey commented:
Yep, you do want to call `build_site()` in this PR. GitHub just directly serves the contents of the `docs/` sub-folder as a (static) website, so the onus is on us to rebuild the website there as needed (and I think it makes sense to do that as part of the PR)

kevinushey commented on vignettes/terminal.Rmd:
```diff
@@ -0,0 +1,100 @@
+---
+title: "Terminal Interaction"
```
Can we use `Interacting with Terminals` as the title here? (For symmetry with the other titles which also are of the form `Interacting with <...>`)

kevinushey commented on vignettes/terminal.Rmd:
```diff
@@ -0,0 +1,100 @@
+
+Each terminal session has a unique **terminal identifier**, a required argument for most of the functions. A terminal identifier is generated and returned when a terminal is created via `terminalCreate()` or `terminalExecute()`, and identifiers of existing terminals can be obtained via `terminalList()` or `terminalVisible()`.
+
+Note: In the current implementation, the terminal identifier is the same as the terminal caption. Thus if a terminal's caption is renamed using the RStudio user interface, the terminal identifier will change without the code knowing about it. This limitation will be fixed in a future release, and the identifier and the caption will no longer be the same; don't write code that assumes the identifi ...
```
Is this something you plan on changing for v1.1, or is that not likely to land until v1.2? If we plan to get this for v1.1 we might want to just omit this clause (or we can just make a note to ensure we update the article in v1.2 or later when this gets in)

kevinushey commented:
LGTM barring one minor comment + a question.

gtritchie commented:
* Changed title to "Interacting with Terminals".
* Removed paragraph about caption and ID being the same, have a PR out for fixing that in 1.1.
* Bumped up `added in` value for all terminal APIs to current preview build (1.1.350). There have been quite a few fixes since 1.1.305.
